components/text_utilities/text_utilities.css

```diff
@@ -0,0 +1,21 @@
+@import url("../../default.css");
+
+.text-center{
+    text-align: center;
+}
+
+.text-left {
+    text-align: start;
+}
+
+.text-right {
+    text-align: end;
+}
+
+.text-small{
+    font-size: 13px;
```

Comment:
maybe you may use rem here for responsiveness too

Reply:
Hey Srishiti, thanks for reviewing, I will change that.

Reply:
Hey, thanks for reviewing, I will change that.